app/scripts/modules/amazon/src/pipeline/stages/bake/bakeStage.html

```diff
@@ -6,6 +6,14 @@
     <stage-config-field label="Regions">
       <checklist items="regions" model="stage.regions" inline="true" include-select-all-button="true"></checklist>
     </stage-config-field>
+    <stage-config-field label="Skip Region Detection">
+      <div class="checkbox" style="margin-bottom: 0">
+        <label>
+          <input type="checkbox" ng-model="stage.skipRegionDetection" />
+          Ensure bake happen only on selected regions.
```

Comment:
```suggestion
          Only bake explicitly selected regions
```

Reply:
Thanks for give it a look, i rewrite it.